src/pycountry/__init__.py

```diff
@@ -48,13 +50,13 @@ def remove_accents(input_str: str) -> str:
     return output_str
 
 
-class ExistingCountries(pycountry.db.Database):
+class ExistingCountries(pycountry.db.Database[pycountry.db.Country]):
     """Provides access to an ISO 3166 database (Countries)."""
 
-    data_class = pycountry.db.Country
+    factory = pycountry.db.Country
     root_key = "3166-1"
 
-    def search_fuzzy(self, query: str) -> List[Type["ExistingCountries"]]:
+    def search_fuzzy(self, query: str) -> List[pycountry.db.Country]:
```

Comment:
`Type[class]` means the class itself (good for factories) not an instance of the class. This also should not return itself, but the underlying data.